meson.build

```diff
@@ -121,45 +126,42 @@ endif
 # Don't add to deps, only link with the binary and the test programs
 # not the shared library.
 
-freetype2_dep = dependency('freetype2', required: false)
-if not freetype2_dep.found()
-  freetype2_dep = cc.find_library('freetype2', has_headers: 'freetype.h', required: false)
-  if not freetype2_dep.found()
-    freetype2_dep = dependency(
-      'freetype2',
-      required: true,
-      fallback: 'freetype2',
-      default_options: ['default_library=static'])
-  endif
-endif
-deps += freetype2_dep
-
-libogg_dep = dependency('ogg', required: false)
-if not libogg_dep.found()
-  libogg_dep = cc.find_library('ogg', has_headers: 'ogg.h', required: false)
-  if not libogg_dep.found()
-    libogg_dep = dependency(
-      'ogg',
+freetype_use_embedded = false
+sdl2_ttf_dep = dependency('SDL2_ttf', required: false)
+if not sdl2_ttf_dep.found()
+  sdl2_ttf_dep = cc.find_library('SDL2_ttf', has_headers: 'SDL_ttf.h', required: false)
+  if not sdl2_ttf_dep.found()
+    # This will use the fallback
+    sdl2_ttf_dep = dependency(
+      'sdl2_ttf',
       required: true,
-      fallback: 'ogg',
-      default_options: ['default_library=static'])
+      fallback: 'sdl2_ttf',
+      default_options: ['default_library=static']
+      )
+    freetype_use_embedded = true
   endif
 endif
-deps += libogg_dep
+deps += sdl2_ttf_dep
 
-vorbis_dep = dependency('vorbis', required: false)
-if not vorbis_dep.found()
-  vorbis_dep = cc.find_library('vorbis', has_headers: 'vorbisfile.h', required: false)
-  if not vorbis_dep.found()
-    vorbis_dep = dependency(
-      'vorbis',
-      required: true,
-      fallback: 'vorbis',
-      default_options: ['default_library=static'])
+# These are required by the SDL2_ttf meson wrap, but not by NetPanzer
```

Comment:
A bit unrelated to this patch... but when updating the appimage script, I noticed freetype being compiled in.